assets/javascripts/modules/toggle.js

```diff
@@ -112,7 +112,7 @@ var toggleEvent = function ($elem) {
       if (event.which === 1 || 32) {
         var $input = $(event.target)
```

Comment:
@feedmypixel I've left a comment on that issue about this, but if it's outside the scope of this PR can you remove this comment? It just adds confusion.